src/test/CMakeLists.txt

```diff
@@ -306,9 +309,11 @@ target_precompile_headers(hyriseTest PRIVATE
     [["storage/table.hpp"]]
     [["types.hpp"]]
 )
+gtest_discover_tests(hyriseTest WORKING_DIRECTORY ${PROJECT_SOURCE_DIR} DISCOVERY_TIMEOUT 60)
```

Comment:
Quite a large timeout (default is 5). Have you run into issues here?

Reply:
Yes, the hyrise binary was timing out frequently for me on a fairly fast Apple M2. 60 sec might be too much though

Reply:
I am fine with 60s, but can you add a comment that the default timed out on a M2?

Reply:
That means the CMake is now significantly slower even a recent M2 chip, yes?

Reply:
I forgot to mention: in order to discover the tests, cmake needs to run the test binary with the `—gtest_list_tests` option. This is what can timeout.

Reply:
Can you add a comment?